### DISCUSSION TITLE
Something in the snippet

### DISCUSSION THREAD

**Discussion by Eneswar** [artisticat1/obsidian-latex-suite#155] [Q&A] [answered]
Hey, something in the snippert turns "ln" into "\ln" and I have searched, manually looked for every single line but I have been unable to find it, can anyone help?

**Comment by candytripn** [comment]
been losing my mind looking for this

**Comment by artisticat1** [comment, marked as answer]
Under _Advanced snippet settings > Snippet variables_ in the plugin settings, remove "ln" from "${SYMBOL}". Apologies for the confusion; let me know if there's any way this could be made clearer

**Reply by Eneswar** [reply]
Awesome, that setting wasnt there before was it? I think I missed it somehow lol.
